.gitlab/llnl-ci-conda.yml

```diff
@@ -42,90 +40,29 @@ conda-setup:
     - ${CONDA_WORK_DIR}/miniconda-${CONDA_ARCH}.sh -b -p ${CONDA_WORK_DIR}/miniconda3-${CONDA_ARCH}
     - *finalize
 
-conda-env-setup:
+.conda_env_setup:
   stage: setup
-  parallel:
-    matrix:
-      - HOST : [ "quartz" ]
-        CONDA_ENV_NAME: [ "quartz-env" ]
-        RUNNER_TYPE: "shell"
-        CONDA_ARCH: "x86_64"
-      - HOST : [ "ruby" ]
-        CONDA_ENV_NAME: [ "ruby-env" ]
-        RUNNER_TYPE: "shell"
-        CONDA_ARCH: "x86_64"
-      # - HOST : [ "lassen" ]
-      #   ENVIRONMENT: [ "BATCH" ]
-      #   CONDA_ENV_NAME: [ "lassen-env" ]
-      #   RUNNER_TYPE: "batch"
-        # CONDA_ARCH: "ppc64le"
-  tags:
-    - $HOST
-    - $RUNNER_TYPE
   script:
     - test -d ${CONDA_WORK_DIR}/${CONDA_ENV_NAME} && exit 0
     - source ${CONDA_WORK_DIR}/miniconda3-${CONDA_ARCH}/etc/profile.d/conda.sh
     - eval "$(conda shell.posix hook)"
-    - conda update -y -n base -c defaults conda
+    - conda update -y -n base -c conda-forge conda
```

Comment:
I think we don't need this line, since this is done for base env, which is common for machines that will use this `miniconda3-${CONDA_ARCH}`, would interfere with each other